lib/child-writer/index.js

```diff
@@ -113,25 +113,36 @@ exports.write = (image, drive, options) => {
     emitter.emit('error', error);
   };
 
-  ipc.server.on('error', emitError);
-  ipc.server.on('message', (data) => {
-    let message = null;
-
+  /**
+   * @summary Bridge robot message to the child writer caller
+   * @function
+   * @private
+   *
+   * @param {String} message - robot message
+   *
+   * @example
+   * bridgeRobotMessage(robot.buildErrorMessage(new Error('foo')));
```

Comment:
Presumably `message` commands will be much more common than `error` commands? To reflect that more common case, perhaps this example should be modified?